code/_onclick/hud/alien.dm

```diff
@@ -1,102 +1,128 @@
-/datum/hud/alien/New(mob/living/carbon/Xenomorph/owner)
+/obj/screen/alien
+	icon = 'icons/mob/screen/alien.dmi'
+
+/obj/screen/alien/Click()
+	if(!isxeno(usr))
```

Comment:
Wouldn't it make more sense to return false here and true if you can click?

Reply:
Yea, but somehow this is how it's been done on tgcode for whatever reason. Nevertheless I'll apply the suggestion.